bin/test-workflow/start

```diff
@@ -26,25 +25,27 @@ Usage: ./start [options]:
                                 - Defaults to 'gke'
                                 - Supports 'jenkins'
                               All other selections are rejected
-    -c, --ci-apps             Run the E2E workflow against the subset of apps tested in CI
+    -c, --ci-apps             Run the E2E workflow against the subset of apps tested in CI.
                               These include:
                                 - summon-sidecar
                                 - secretless-broker
                                 - secrets-provider-init
                                 - secrets-provider-standalone
-    -a, --apps <app>[,<app>]  Specify which test apps to install
-                              Multiple apps should be specified as a single comma-delimited string
-                              Excluding this flag runs all supported apps
-                              Currently supports:
+    -a, --apps <app>[,<app>]  Specify which test apps to install.
+                              Multiple apps should be specified as a single comma-delimited string.
+                              Excluding this flag runs all supported 'authn-k8s' apps (if '--jwt'
```

Comment:
Should this be "This defaults to the supported  apps for 'authn-k8s'" or something like that?

Reply:
Yes, that's probably clearer. I added your suggested change.
I also made another small change to make sure that when `--jwt` flag is selected, the default apps to get deployed are all of the JWT-authentication based applications.